pyproject.toml

```diff
@@ -27,11 +27,11 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 dependencies = [
-    "numpy",
-    "pyarrow",
-    "pint",
-    "loguru",
-    "rich",
+    "numpy==1.26.4",
```

Comment:
Can we loosen the patch number on all these so that we can get bug fixes?
```
pyarrow~=15.0.2
```

Reply:
Sure.